tasks/main.yml

```diff
@@ -21,9 +21,9 @@
   when: not elasticsearch_installed.stat.exists
 
 - name: Install ES plugins
-  shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}remove {{ item }} && /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
-  when: head_plugin_path.stat.exists != true
+  shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
   with_items: "{{ plugins_to_install }}"
+  ignore_errors: True
```

Comment:
The simplest way is probably

```yaml
- name: Install ES plugins
  shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
  with_items: "{{ plugins_to_install }}"
  register: plugin_response
  failed_when: "'ERROR' in plugin_response.stdout and 'uninstall it first' not in plugin_response.stdout"
```
'uninstall it first' is just a sample string that will almost certainly only be in that error. What do you think?